File: Python/String-2/count_hi.py
"""
Return the number of times that the string "hi" appears anywhere in the given string.

count_hi('abc hi ho') → 1
count_hi('ABChi hi') → 2
count_hi('hihi') → 2

@author unobatbayar
"""

def count_hi(str):
  count = 0
  for x in range(0, len(str)-1):
    if str[x] == 'h'and str[x+1] == 'i':
        count += 1
  return count

File: Java/String-2/countHi.java
/*

Return the number of times that the string "hi" appears anywhere in the given string.

countHi("abc hi ho") → 1
countHi("ABChi hi") → 2
countHi("hihi") → 2

*/
class countHi{
	public int countHi(String str) {
	Integer counter = 0;
  
	for (int i = 0; i <str.length()-1; i++){
		if (str.substring(i, i + 2).equals("hi")){
        		counter = counter + 1;
        	}
        }
        return counter;
	}	


}
